### ISSUE TITLE
Novation Summit

### ISSUE THREAD

**Issue by christofmuc** [christofmuc/KnobKraft-orm#112]
seems by know to have been documented in the Sequencer forum. 

Here is one link: 
https://gearspace.com/board/electronic-music-instruments-and-electronic-music-production/1149964-novation-peak-96.html

**Comment by christofmuc** [comment]
There is by now a very detailed analysis of the Novation Summit:
https://gearspace.com/board/showpost.php?p=15306436&postcount=1566

This will make it fairly easy. I have prototyped an adaptation, should be close to done.

**Comment by christofmuc** [comment]
This was released. Waiting on user feedback.

**Comment by dromer** [comment]
@christofmuc finally built KnobKraft and tested!

First issue I see is that importing Bank D actually imports Bank A, but I can open a ticket for that if you want ;)

**Comment by christofmuc** [comment]
@dromer Cool, that's better than nothing! We can use this tickets for details!

**Comment by christofmuc** [comment]
@dromer I actually had a bug in the calculation of the patch number. I just released KnobKraft 2.0, which contains a hopefully fixed version for the Summit!

**Comment by dromer** [comment]
@christofmuc thnx! I'll give it a try when I have time :)

**Comment by RadekPilich** [comment]
Hi, this was probably never fixed fully. I'm kindly asking for a fix and then I can retest.

In the most recent versions of Summit and KK, the "In synth" Bank A pulls nothing, Bank B pulls Bank A, Bank C pulls Bank B, Band D pulls Bank C.

**Comment by christofmuc** [comment]
Cool, this looks like a simple +1/-1 bug in the adaptation code itself. I opened a new issue for that, that should be easily (?) fixable.
